examples/examples_requirements.txt

```diff
@@ -1,4 +1,5 @@
 astra-toolbox
+cached_property
```

Comment:
Perhaps a brief mention in the example requirements section to explain why this is (hopefully temporarily) included as a dependency?

Reply:
You mean add a comment in the example_requirements.txt  or add a sentence in the docs?

Reply:
On consideration, perhaps the best approach is in fact to just create an issue reminding us to remove this once the `xdesign` issue is resolved.

Reply:
Created as #160.